cli/pkg/providers/aws/aws.go

```diff
@@ -51,6 +52,17 @@ func (p *provider) GetOverrides() tmpl.Variables {
 	}
 }
 
+func (p *provider) IsEnvironmentValid() (bool, error) {
```

Comment:
You can just return `error`.

```go
if err != nil { ... }
```

If error is not nil this means that the environment is not valid, the `bool` value is redundant